airsenal/framework/utils.py

```diff
@@ -1259,6 +1259,26 @@ def get_recent_playerscore_rows(
     return rows[-num_match_to_use:]
 
 
+def get_playerscore_for_player_gameweek(
```

Comment:
I think this won't do what we want for double gameweeks, we probably want to return the list of all player score rows for a gameweek here, and then compute total points etc. elsewhere (could also just return total points and minutes from this function).

Reply:
fixed this